### PR TITLE
docs(dropdown): decompose manual triggering demo

### PR DESCRIPTION
**Pull request by YevheniiaMazur**
# PR Checklist
 - [x] built and tested the changes locally.
 - [x] updated API documentation.
 - [x] updated demos.

Closes: https://github.com/valor-software/ngx-bootstrap/issues/4110
What was done:
- [x] demo was decomposed into two parts:
2partsoftrgg
- [x] was used direct change of `isOpen` property instead of creating special method for it:
directchange
- [x] API documentation was updated:
apidoc

### CODE REVIEW

File: demo/src/app/components/+dropdown/demos/trigger-by-isopen-property/trigger-by-isopen-property.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  </button>
+  <ul *dropdownMenu class="dropdown-menu" role="menu">
+    <li role="menuitem"><a class="dropdown-item" href="#">Action</a></li>
+    <li role="menuitem"><a class="dropdown-item" href="#">Another action</a>
+    </li>
+    <li role="menuitem"><a class="dropdown-item" href="#">Something else
+      here</a></li>
+  </ul>
+</div>
+<button type="button" class="btn btn-primary" (click)="dropdown.isOpen=!dropdown.isOpen">Toggle</button>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/demos/trigger-by-isopen-property/trigger-by-isopen-property.html]: `dropdown.isOpen = !dropdown.isOpen`

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 2]
```diff
@@ -72,17 +73,26 @@ export const demoComponentContent: ContentSection[] = [
...
         html: require('!!raw-loader?lang=markup!./demos/triggers-manual/triggers-manual.html'),
-        description: `<p>Dropdown can be triggered in two ways:
-          <ul>
-            <li>by toggling <code>isOpen</code> property</li>
-            <li>by <code>show</code>/<code>hide</code> methods from directive</li>
-          </ul></p>`,
+        description: `<p>Dropdown can be triggered by <code>show</code>, <code>hide</code> and
+          <code>toggle</code> methods from directive
+          <br>
+          Use method <code>toggle(true)</code> if you want create toggler, or <code>toggle(false)</code>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: if you want create toggler -> if you want to toggle the dropdown

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 3]
```diff
@@ -72,17 +73,26 @@ export const demoComponentContent: ContentSection[] = [
...
+          if you want only close opened dropdown.
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: if you want only close opened dropdown -> if you only want to close the dropdown

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 4]
```diff
@@ -72,17 +73,26 @@ export const demoComponentContent: ContentSection[] = [
...
+          </p>`,
         outlet: DemoDropdownTriggersManualComponent
       },
+      {
+        title: 'Trigger by isOpen property',
+        anchor: 'trigger-by-isopen-property',
+        component: require('!!raw-loader?lang=typescript!./demos/trigger-by-isopen-property/trigger-by-isopen-property.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/trigger-by-isopen-property/trigger-by-isopen-property.html'),
+        description: `<p>Dropdown can be shown or hidden by direct change of <code>isOpen</code> input property</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: by direct change of -> by changing

File: demo/src/ng-api-doc.ts [hunk 5]
```diff
@@ -1802,7 +1802,7 @@ export const ngdoc: any = {
       },
       {
         "name": "toggle",
-        "description": "<p>Toggles an element’s popover. This is considered a “manual” triggering of\nthe popover.</p>\n",
+        "description": "<p>Toggles an element’s popover. This is considered a “manual” triggering of\nthe popover. With parameter true allow toggling, with parameter false\nonly hide opened dropdown. Parameter usage will be removed in 3rd ngx bootstrap release</p>\n",
```
Comment (IlyaSurmay) [inline demo/src/ng-api-doc.ts]: allow -> allows
hide -> hides
also put `true` in `<code></code>`
3rd ngx bootstrap release  -  ngx-bootstrap v3

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 6]
```diff
@@ -72,17 +73,26 @@ export const demoComponentContent: ContentSection[] = [
...
+          if you want to only close opened dropdown.
+          </p>`,
         outlet: DemoDropdownTriggersManualComponent
       },
+      {
+        title: 'Trigger by isOpen property',
+        anchor: 'trigger-by-isopen-property',
+        component: require('!!raw-loader?lang=typescript!./demos/trigger-by-isopen-property/trigger-by-isopen-property.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/trigger-by-isopen-property/trigger-by-isopen-property.html'),
+        description: `<p>Dropdown can be shown or hidden by changing of <code>isOpen</code> input property</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: changing of -> changing

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**